.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "type": "shell",
+      "command": "npx tsc --noEmit --pretty | grep -A 1 \"is not assignable to parameter of type 'MessageKeys\"",
```

Comment:
Running this task I get a lot of errors show like: 
```
src/features/common/InputTypes/AutoCompleteCountry.tsx:69:23 - error TS2345: Argument of type 'string' is not assignable to parameter of type 'MessageKeys<{ auto: string; ac: string; af: string; ax: string; al: string; dz: string; as: string; ad: string; ao: string; ai: string; aq: string; ag: string; ar: string; am: string; aw: string; au: string; at: string; az: string; ... 241 more ...; zw: string; }, "de" | ... 258 more ... | "zw">'.
```

So is the plan to fix them after merging this or within this PR?